compiler/qsc_hir/src/visit.rs

```diff
@@ -65,6 +65,7 @@ pub fn walk_item<'a>(vis: &mut impl Visitor<'a>, item: &'a Item) {
         ItemKind::Callable(decl) => vis.visit_callable_decl(decl),
         ItemKind::Namespace(name, _) => vis.visit_idents(name),
         ItemKind::Ty(name, _) => vis.visit_ident(name),
+        ItemKind::Export(_, _) => {}
```

Comment:
here too, it should probably visit the ident:

```suggestion
        ItemKind::Export(name, _) => vis.visit_ident(name),
```